pygrametl/tables.py

```diff
@@ -1843,7 +1843,7 @@ class BatchFactTable(FactTable):
     """
 
     def __init__(self, name, keyrefs, measures=(), batchsize=10000,
-                 targetconnection=None):
+                 usevalues=False, targetconnection=None):
```

Comment:
I don't think `usevalues` is a good name since the INSERT statement always contains "VALUES" no matter if it inserts one row or many rows. Could it be `usemanyvalues` or `usemultivalues`?